Sources/NIOCore/EventLoop.swift

```diff
@@ -314,7 +314,7 @@ public protocol EventLoop: EventLoopGroup {
     /// This executor can be used to isolate an actor to a given ``EventLoop``. Implementers are encouraged to customise
     /// this implementation by conforming their ``EventLoop`` to ``NIOSerialEventLoopExecutor`` which will provide an
     /// optimised implementation of this method, and will conform their type to `SerialExecutor`. The default
-    /// implementation returns a ``NIODefaultSerialEventLoopExecutor`` instead, which provides suboptimal performance.
+    /// implementation provides suboptimal performance.
```

Comment:
Feels like we're losing info here.  Perhaps that information would be better in a regular comment more than a doc comment though.

Reply:
I think we leaked implementation details here before. `NIODefault...Executor` is `internal` and we shouldn't talk about it in public docs.